mkosi.md

```diff
@@ -287,30 +287,34 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
 
 `Distribution=`
 
-: Matches against the configured distribution. Multiple distributions may
-  be specified, separated by spaces. If multiple distributions are specified,
-  the condition is satisfied if the current distribution equals any of the
-  specified distributions.
+: Matches against the configured distribution. Multiple distributions may be
+  specified, separated by spaces. A single distribution match can be negated by
+  preceding the value with `!`. If multiple distributions are specified, the
+  condition is satisfied if the current distribution equals any of the specified
+  distributions.
 
 `Release=`
 
-: Matches against the configured distribution release. If this condition
-  is used and no distribution has been explicitly configured yet, the
-  host distribution and release are used. Multiple releases may be specified,
-  separated by spaces. If multiple releases are specified, the condition is
-  satisfied if the current release equals any of the specified releases.
+: Matches against the configured distribution release. If this condition is used
+  and no distribution has been explicitly configured yet, the host distribution
+  and release are used. A single release match can be negated by preceding the
+  value with `!`. Multiple releases may be specified, separated by spaces. If
+  multiple releases are specified, the condition is satisfied if the current
+  release equals any of the specified releases.
 
 `PathExists=`
 
 : This condition is satisfied if the given path exists. Relative paths are
   interpreted relative to the parent directory of the config file that the
-  condition is read from.
+  condition is read from. A match can be negated by preceding the path with a
+  `!`.
 
 `ImageId=`
 
 : Matches against the configured image ID, supporting globs. If this condition
   is used and no image ID has been explicitly configured yet, this condition
-  fails. Multiple image IDs may be specified, separated by spaces. If multiple
+  fails.  A single image ID match can be negated by preceding the value with
+  `!`. Multiple image IDs may be specified, separated by spaces. If multiple
```

Comment:
I think we can also get away here by allow negation of the full result but not individual items?